ush/set_rrfs_config_SDL_VDL_MixEn.sh

```diff
@@ -0,0 +1,35 @@
+
+l_both_fv3sar_gfs_ens=.false. #if true, ensemble size is increased with GDAS ensemble (MixEn)
+nsclgrp=1                     #number of scales for scale-dependent localization (SDL)
+ngvarloc=1                    #number of scales for variable-dependent localization (VDL)
+assign_vdl_nml=.false.        #if true, vdl_scale and vloc_varlist are used to set VDL
```

Comment:
@shoyokota Should lines 2,3,4,5 be removed from this file?  Since these options are set in config_defaults.sh, repeating them here overwrites what is set and requires this file to be changed to turn on/off the options.

Reply:
Overwriting these options in set_rrfs_config_SDL_VDL_MixEn.sh is as I expect and removing them is not required. (A lot of options set in config_defaults.sh are already overwritten in set_rrfs_config.sh and set_rrfs_config_general.sh.) If you would like to change these options, please change set_rrfs_config_SDL_VDL_MixEn.sh (not config_defaults.sh).